routes/api/v1/foods.js

```diff
@@ -1,19 +1,86 @@
-
-var express = require("express");
+var express = require('express');
 var router = express.Router();
 var Food = require('../../../models').Food;
 
 /* GET all foods */
-router.get("/", function (req, res) {
+router.get('/', function (req, res) {
   Food.findAll()
     .then(foods => {
-      res.setHeader("Content-Type", "application/json");
-      res.status(200).send(JSON.stringify(foods));
+      if (foods.length === 0) {
```

Comment:
```suggestion
      if (!array.length) {
```
this is equivalent to array.empty? in ruby

Reply:
I like it!